src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class RightTriangle extends Figure {
```

Comment:
The class 'RightTriangle' should not begin with an empty line according to the checklist.